### ISSUE TITLE
git fsck failures with all repos

### ISSUE THREAD

**Issue by y** [vim-scraper/vim-scraper#87]
If you enable the git option `fsckobjects = true`, which is recommended practice, all the vim-scripts repos fail like this:
 
    $ git clone https://github.com/vim-scripts/scratch.vim.git
    Cloning into 'scratch.vim'...
    remote: Counting objects: 6, done.
    remote: Total 6 (delta 0), reused 0 (delta 0), pack-reused 6
    error: object 90c2bc4bfe01151aa5c71d9cb243b05bc105ae01: badTimezone: invalid author/committer line - bad time zone
    fatal: Error in object
    fatal: index-pack failed

**Comment by bronson** [comment]
Yep, sounds like this issue: http://vim-scripts.org/news/2011/06/23/picky-about-timezones.html

Fixing it requires regenerating every repo from the ground up.
